src/benchmark/file_io_read_micro_benchmark.cpp

```diff
@@ -97,7 +105,7 @@ BENCHMARK_DEFINE_F(FileIOMicroReadBenchmarkFixture, READ_NON_ATOMIC_RANDOM)(benc
     std::cout << "open error " << errno << std::endl;
   }
 
-  const auto NUMBER_OF_BYTES = uint32_t{static_cast<uint32_t>(state.range(0) * MB)};
+  const auto NUMBER_OF_BYTES = allign_to_pagesize(state.range(0));
   const auto uint32_t_size = ssize_t{sizeof(uint32_t)};
   const auto read_data_size = NUMBER_OF_BYTES / uint32_t_size;
   const auto max_read_data_size = static_cast<size_t>(read_data_size);
```

Comment:
please have a look at #41, where we moved all of this duplicate code to the .hpp - therefore these changes might no longer be necessary